content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+### reconstruct blown highlights
+
+While a well-exposed image will make post-processing much easier, darktable provides some tools handle blown highlights, in some cases even reconstructing lost colors or structure.
+
+The [_highlight reconstruction_](../../module-reference/processing-modules/highlight-reconstruction.md) module provides a very basic attempt to manage blown highlights. By default, if any color channel is clipped, this module will clip the other channels to white -- this prevents darktable from showing unnatural colors in the highlights. However it can be altered to attempt to reconstruct the lightness or color using any un-clipped channels. While this can re-introduce some detail, beware of the unrealistic colors that can result.
+
+A better alternative is to disable _highlight reconstruction_ and instead use the [_reconstruct tab on the filmic rgb module_](../../module-reference/processing-modules/filmic-rgb.md#reconstruct). As with the _highlight reconstruction_ module, filmic is able to reconstruct detail based on unclipped channels. Where this is not possible, it can also blend blown highlights smoothly with the rest of the image or pull colors from adjacent pixels. When incorporating this step into your workflow, you should do it at the same time as other _filmic rgb_ edits, performing adjustments in each tab of the module in turn, from left to right.
+
+### adjust angle and perspective
+
+The [_rotation and perspective_](../../module-reference/processing-modules/perspective-correction.md) module can be used to adjust the angle of the image or to simulate the functionality of a tilt/shift lens by altering the perspective, making converging horizontal and/or vertical lines parallel (keystone correction). This latter technique is most commonly used for architectural photography. If you just want to correct the angle of the horizon you can do this by right-clicking and dragging along the horizon line.
+
+### remove spots and unwanted objects
+
+Use the [_retouch_](../../module-reference/processing-modules/retouch.md) module to remove unwanted objects by replacing pixels with detail from elsewhere in the image. This module also offers powerful techniques for removing large-scale objects (such as spots or blemishes) while leaving fine-scale details (like hairs and follicles) intact. The most common use for this module is to remove dust spots from images or blemishes from skin.
+
+### remove atmospheric haze
+
+There are two methods for removing atmospheric haze in darktable -- you can either use the [haze removal](../../module-reference/processing-modules/haze-removal.md) module or the "dehaze" preset in the [_diffuse or sharpen_](../../module-reference/processing-modules/diffuse.md) module.
```

Comment:
_dehaze_ is simpler, but _diffuse or sharpen_ can give more flexibility where needed